pytri/js/ColorGraphLayer.js

```diff
@@ -60,10 +60,6 @@ class ColorGraphLayer extends Layer {
         let xPos = nodeData.x - this.xSubtract
```

Comment:
Can we get rid of the subtracts here as well?

Reply:
Blerg. `LargeGraphLayer` != `ColorGraphLayer`. I was editing the wrong file... thanks for your patience!